project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/layout/layout.less

```diff
@@ -12,7 +12,7 @@ img {
     width: 100%;
     min-height: 100%;
 
-    .is-no-flex &, .is-safari & {
+    .is-safari & {
```

Comment:
Is the class `is-safari` still necessary?

Reply:
really dont know ... but will be checked later during refactoring

Reply:
Okay, let's refactor this later.